projects/core/src/product/services/product-page-meta.resolver.ts

```diff
@@ -66,7 +68,7 @@ export class ProductPageMetaResolver
     protected routingService: RoutingService,
     protected productService: ProductService,
     protected translation: TranslationService,
-    @Optional() protected basePageMetaResolver?: BasePageMetaResolver
+    protected basePageMetaResolver?: BasePageMetaResolver
```

Comment:
Why dropping @Optional and `?` in minor?

Reply:
Only the Optional is dropped, seems to be enough to allow for extensibility during minors